### ISSUE TITLE
Find event by date

### ISSUE THREAD

**Issue by thenewtonium** [jsyiek/AU2#57]
Following on from issue #55: the interface for locating an event in `Event -> Update` (and also `Event -> Delete`) gets kind of unwieldy with a lot of events. 

It would be a lot easier to search for an event using its datetime stamp.

At a minimum, the list of events should include the datetime of each event and be sorted by datetime. This is doable within the current plugin framework since `inquirer.List` can have options return different values to their display values, so we just need to change the `options_functions` of the event update export accordingly.

More useful would be a more sophisticated method of selecting an event, for example grouping into sub-lists by date or using something like `Searchable` from PR #56. This would require reworking how plugin exports are handled however.

**Comment by jsyiek** [comment]
Yeah, this would require an entirely different approach to #56 - what if each `option` could take in the results of all previous options?

**Comment by thenewtonium** [comment]
Yeah that does seem like a good approach for grouping by date!

**Comment by thenewtonium** [comment]
I'm not sure if we actually want to implement this anymore, #58 seems to be sufficient and an extra layer (grouping by date or having a search function) might just be more annoying. If we do implement this it should probably somehow be made togglable by `UIConfigPlugin`.
